README.md

```diff
@@ -32,6 +32,27 @@ The different alerting rules in this repository are:
 
 * [**Alert Rule QuotaExceeded**](https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/o11y/alert-rule-QuotaExceeded.md)
 
+### SLO Alerts
+
+SLO (Service Level Objective) alert rules are rules defined to monitor and alert when a service or system is not meeting its specified service level objectives.
```

Comment:
Let's keep this line and the ones below to the same length limit as on the rest of the file.

Reply:
updated.